Update rds.md (#22838)
DataDog/documentation: content/en/database_monitoring/setup_postgres/rds.md

## Install the RDS Integration

> To see infrastructure metrics from AWS such as CPU etc alongside the database telemtryin DBM, install the [RDS integration][13] and DBM will correlate automatically (optional).

Reviewer: ```suggestion
To see infrastructure metrics from AWS, such as CPU, alongside the database telemetry in DBM, install the [RDS integration][13] (optional).
```
